virtual_ship/sailship.py

```diff
@@ -203,10 +203,11 @@ def sailship(config: VirtualShipConfiguration):
     print("Simulating onboard ADCP.")
     simulate_adcp(
         fieldset=config.adcp_fieldset,
-        out_file_name=os.path.join("results", "adcp.zarr"),
+        out_path=os.path.join("results", "adcp.zarr"),
         max_depth=config.ADCP_settings["max_depth"],
         min_depth=-5,
-        bin_size=config.ADCP_settings["bin_size_m"],
+        num_bins=(-5 - config.ADCP_settings["max_depth"])
```

Comment:
so, before the argument was bin_size. I did not like that because it's very depending on the min and max. now you just choose the number of bins, and it doesn't break if you change min/max

Reply:
to keep the behaviour in sailship the same we apply this small conversion formula. for now until we rework the config file

Reply:
@ammedd, what is the setting on an actual ADCP? I think we should keep as close as possible to what is done in the real instruments

Reply:
This has been on my Wishlist (https://github.com/OceanParcels/Virtual_ship_classroom/issues/24) and is more realistic, so go ahead